autoload/ale/fixers/appleswiftformat.vim

```diff
@@ -0,0 +1,47 @@
+" Author: (bosr) <bosr@bosr.cc>
+" Description: Integration of apple/swift-format formatter with ALE.
+
+call ale#Set('swift_appleswiftformat_executable', 'swift-format')
+call ale#Set('swift_appleswiftformat_use_swiftpm', 0)
+
+function! ale#fixers#appleswiftformat#GetExecutable(buffer) abort
```

Comment:
This function is exactly the same used in the linter. Better move it to *autoload/ale/swift.vim* file to keep the code DRY.